src/js/actions/index.js

```diff
@@ -36,16 +37,28 @@ export function hideSpinner() {
   };
 }
 
-export function showModal() {
+export function showModal(modalName) {
+  if (modalName === 'locationChooser') {
+    return {
+      type: TOGGLE_LOCATION_CHOOSER,
+      payload: true,
+    };
+  }
   return {
```

Comment:
I prefer this to having its own action for better readability. (perhaps we could have 'showLocationChooser' and 'showUserActions' to distinguish both)